acceptance/openstack/compute/v2/compute.go

```diff
@@ -802,8 +802,13 @@ func CreateServerWithPublicKey(t *testing.T, client *gophercloud.ServiceClient,
 // CreateVolumeAttachment will attach a volume to a server. An error will be
 // returned if the volume failed to attach.
 func CreateVolumeAttachment(t *testing.T, client *gophercloud.ServiceClient, blockClient *gophercloud.ServiceClient, server *servers.Server, volume *volumes.Volume) (*volumeattach.VolumeAttachment, error) {
+	tag := tools.RandomString("ACPTTEST", 16)
+	dot := false
+
 	volumeAttachOptions := volumeattach.CreateOpts{
-		VolumeID: volume.ID,
+		VolumeID:            volume.ID,
+		Tag:                 tag,
+		DeleteOnTermination: dot,
```

Comment:
The client's microversion will need to be set to 2.79 in `acceptance/openstack/compute/v2/volumeattach_test.go`.

Reply:
:) fixed!